docs/software/webserver_stack.md

```diff
@@ -504,6 +504,9 @@ It is designed to be secure, fast, standards-compliant, and flexible while being
 
     After the installation, check if Lighttpd service is running on `http://<your.IP>`.
 
+    How much time do you have to set up Lighttpd ? 5 minutes? 10 minutes? or more ? Find here a quick start documentation: [link](https://redmine.lighttpd.net/projects/1/wiki/TutorialConfiguration).
+
+
```

Comment:
Not sure if this makes sense in our case, where everything is set up already? Could confuse when readers think they need to follow this linked tutorial after having Lighttpd installed though `dietpi-software`. It even includes the creation of a daemon service, which is not required at all on any Debian install, moreover it shows a SysV daemon setup where Debian and DietPi use systemd, hence this would conflict.

Reply:
ok. let's discuss on Tuesday. I will remove my commit by then.